.github/workflows/test_common.yml

```diff
@@ -84,7 +76,21 @@ jobs:
       run: |
         echo "## Run Information" >> $GITHUB_STEP_SUMMARY
         echo "- **Node1**: ${{ env.NODE_1 }}" >> $GITHUB_STEP_SUMMARY
-        echo "- **Node2**: ${{ env.NODE_2}}" >> $GITHUB_STEP_SUMMARY
-        echo "- **Additonal Nodes**: ${{ env.ADDITIONAL_NODES}}" >> $GITHUB_STEP_SUMMARY
+        echo "- **Node2**: ${{ env.NODE_2 }}" >> $GITHUB_STEP_SUMMARY
+        echo "- **Additonal Nodes**: ${{ env.ADDITIONAL_NODES }}" >> $GITHUB_STEP_SUMMARY
         echo "## Test Results" >> $GITHUB_STEP_SUMMARY
         echo "Allure report will be available at: https://waku-org.github.io/waku-interop-tests/${{ github.run_number }}" >> $GITHUB_STEP_SUMMARY
+        {
+          echo 'JOB_SUMMARY<<EOF'
+          cat $GITHUB_STEP_SUMMARY
+          echo EOF
+        } >> $GITHUB_ENV

```

Comment:
We should add ${{ job.status }} as well to see if the job is failed or success

Reply:
job status added at https://github.com/waku-org/waku-interop-tests/pull/13/commits/46cfe0152760acf1b5a0ac6709b922df84e13d01

Let me know, about other things you would like to change. I want matching colors and shorted node description in notification. :)

Reply:
Looks much better
Would be great to have the user who triggered the workflow. And also the Event. 
But this is nice to have

Reply:
However the allure links do not work, I'm looking into it right now

Reply:
Seems unrelated to this PR